frontend/awx/views/jobs/hooks/useRelaunchJob.tsx

```diff
@@ -90,6 +90,7 @@ export function useRelaunchJob(jobRelaunchParams?: JobRelaunch) {
         variant: 'danger',
         title: t('Failed to relaunch job'),
         children: error instanceof Error && error.message,
+        timeout: true,
```

Comment:
This should be a number representing the miliseconds that the toast should be visible on the screen.

Reply:
So I saw setting the timeout to `true` default the timeout to 8000ms. Do we have a standard timeout duration for toasts?

Reply:
Did was not aware of an default timeout for the toast, though I do think that 8000ms is too long.  A user could potentially relaunch several jobs at once, or in succession and their screen could quickly get filled with toasts if they all fail.  Lets go with a shorter timeout.  2000 ms seems comfortable to me.